data/changelog.txt

```diff
@@ -3660,731 +1862,89 @@ Overmap saves moved to json.
-JSON reorganization
-Move items from melee.json to generic.json
-Extend visitable interface
-Add quantity parameter to item constructor
-Implement visitable *::has_item()
-Deprecate unused LEAK_RAD flag
-Remove unnecessary tool values from fake weapons
-Separate the logic of use_function and iuse_actor.
-Generic martial art bonuses
-Turrets use fake weapons
-Remove player.h from monster.h and dialogue.h
-Slightly cleaner way to handle ifdeffed code.
-Check more carefully for matching ammo
-Improve handling and display of reload times
-Don't require fake ammo type for UPS-only weapons
-Normalize legacy saves to charges >= 0
-JSONized mutation encumbrance
-Extract coordinate conversion functions to their own module.
-Drop now invalid casing property
-Encapsulate contents and charges (part 8)
-Implement Lua wrapper to store and use either value or reference objects in Lua
-Move conversion kits and memory cards to generic.json
-Overhaul gun noise
-Move itms from melee.json to generic.json
-Provide filters for (un)setting ammo
-Improve reloading messages
-Include for round()
-Remove unused mapgen code
-JSON survival basement and zombies
-Allow iuse actors to describe themselves
-Move some items from melee.json to generic.json
-Drop unused parameter from visitable class
-Encapsulate pain
-Change iexamine signature
-Implement visitable::remove_items_with
-Remove ATTACKMON from monsters that don't need it
-De-uglify code for wearing headgear with power armor
-Make DIG iuse do something useful
-Renamed car_corner_aban1 from green to car corner
-Remove null material from item definitions.
-Provide default for reload time
-Static helpers
-JSONize mutation equipment restrictions
-Move code to handle json entity inheritance into item_factory.
-Item variants with abritrary order
-Deprecate it_tool
-Move function print_menu_items from game class to anon namespace
-Deprecate it_comest
-Move more methods to visitable interface
-Create single visibility_variables cache in map class
-Move more light sources to lighting.json
+Remove hardcoded moves cost when unloading from containers.
+Weather Reader reworked.
+Adds Prying quality, adds helper functions for quality checks.
+Better NPC complaints.
+Implements magazine wells.
+Customizable bite attacks.
+Add inheritance, strict and bounds-checking to MONSTER.
+Allergy flags instead of allergy materials.
+Move turret monattacks to a json actor.
+Vehicle weight cached, more realistic.
+Implement Lua wrapper to store and use either value or reference objects in Lua.
 Create Blank Building Template.txt
-Comestibles support inheritance
-Correctly apply temperature modifiers
-Visual check for ABSORBS message.
-Add contexts to scenarios in json string extractor
-Add is_blind function and replace checks
-Only set ammo on mapgen for tool, gun or magazine
-Rework selection of NPC weapons
-Core functionality of bionic slots system (again)
-Ensure vehicle parts category is used
-Overhaul vehicle part installation
```

Comment:
If I knew what this was, I would suggest not deleting it.

Reply:
Good catch, that's the super popular, "Add lifting and jacking requirements to vehicle modification."